extensions/cql/standard/clause_2_conformance.adoc

```diff
@@ -32,14 +32,14 @@ binding between this the <<rc_filter,Filter>> conformance class the
 <<OAFeat-1,OGC API - Features - Part 1: Core>> standard.
```

Comment:
there is an "and" missing between "this the"

Reply:
Added "and".